assets/src/components/inactiveNotificationModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext } from "react"
+import { StateDispatchContext } from "../contexts/stateDispatchContext"
+import { closeIcon } from "../helpers/icon"
+import { Notification } from "../realtime.d"
+import { setNotification } from "../state"
+import { title } from "./notifications"
+
+const InactiveNotificationModal = ({
+  notification,
+  removeNotification,
+  setInactiveNotification,
+}: {
+  notification: Notification
+  removeNotification: (id: number) => void
+  setInactiveNotification: (notification: Notification | null) => void
```

Comment:
Some places (like here) this is `inactiveNotification` and other places (like css) it's `invalidNotification`. Let's standardize on one.